spark/pom.xml

```diff
@@ -36,10 +36,17 @@
 
   <properties>
     <spark.version>1.4.1</spark.version>
-    <scala.version>2.10.4</scala.version>
-    <scala.binary.version>2.10</scala.binary.version>
   </properties>
 
+  <profiles>  
+    <profile>
+      <id>scala-2.11</id>    
+      <properties>
+        <spark.version>2.0.0-SNAPSHOT</spark.version>
```

Comment:
not sure that would work - we could need the artifact in maven central repo or similar to build, and snapshot builds are not published..